From ee054e5ad42717697f6b1b87c36e6f4656a008b6 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Wed, 24 Jan 2024 08:52:26 -0600
Subject: [PATCH] link against dynamic boost test lib

---
 test-suite/quantlibtestsuite.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test-suite/quantlibtestsuite.cpp b/test-suite/quantlibtestsuite.cpp
index b21ac6d7a..d46a56315 100644
--- a/test-suite/quantlibtestsuite.cpp
+++ b/test-suite/quantlibtestsuite.cpp
@@ -20,6 +20,7 @@
 
 #include <ql/qldefines.hpp>
 
+#define BOOST_TEST_DYN_LINK
 #define BOOST_TEST_MODULE QuantLibTests
 
 #ifdef QL_ENABLE_PARALLEL_UNIT_TEST_RUNNER
-- 
2.42.0

